src/PrestaShopBundle/Resources/config/routing/admin/sell/orders/carts.yml

```diff
@@ -66,3 +67,21 @@ admin_carts_set_free_shipping:
     cartId: \d+
   options:
     expose: true
+
+admin_carts_add_rule:
+  path: /{cartId}/rules/add
+  methods: [POST]
+  defaults:
+    _controller: PrestaShopBundle:Admin/Sell/Order/Cart:addCartRuleToCart
+    _legacy_controller: AdminCarts
+  options:
+    expose: true
+
+admin_carts_delete_rule:
```

Comment:
just got an idea 🤔 shall we put "ajax" in routes handling AJAX requests ?
it's not pretty but this would provide a simple way for module developers to find out which route is an ajax route and which is not

not sure about that ... wdyt @PierreRambaud @jolelievre @matthieu-rolland @Progi1984 ?